### DISCUSSION TITLE
Whether or not Admin approval is required when creating an event

### DISCUSSION THREAD

**Discussion by ngnlyoosun** [pacemakerIT/Gachi#13] [Q&A]
Should events submitted by mentors be displayed on the site immediately, or should they first go through an evaluation and approval process by an administrator to ensure their appropriateness before being published?

Ralph said he wanted communication and networking to flow smoothly between people without him, but if every event required admin approval, Ralph would have a lot to worry about.

**Comment by nhistory** [comment]
I believe it would be best to have events submitted by mentors go through an evaluation and approval process by an administrator before being published on the site. While it's important for communication and networking to flow smoothly between people, having every event require admin approval might be too much for Ralph to handle. However, having multiple staff members with access to the site's admin account can help distribute the workload and ensure that all content is appropriately controlled and managed in the early stages.
